docs/features/runs/1_Creating Runs/2_Adv Runs/3_creating-runs-from-config.md

```diff
@@ -45,6 +45,7 @@ compute:
       MY_ENVIRONMENT_VARIABLE: "example"
 ```
 
+<!-- As an aside: We need to add e2e tests around the config file experience -->
```

Comment:
Reminder to remove this prior to merging. @luca3rd have you created a ticket for this? If not I can just make a more complicated config example on https://github.com/PyTorchLightning/grid-tutorials that potentially can be used?

Reply:
Actually this aside does not appear on local build. We can disregard it.